### DISCUSSION TITLE
Como eu consigo trocar o literal "Outras ações" do PoPageDynamicSearch?

### DISCUSSION THREAD

**Discussion by HandreiFont** [po-ui/po-angular#1672] [Angular] [answered]
Boa tarde,
estou utilizando o componente po-page-dynamic-search e gostaria de trocar o literal de outras ações
![image](https://user-images.githubusercontent.com/107151602/229227953-0bdb3a68-3958-41cc-a2c0-c6ddbf05f769.png)
mas como essa propriedade vem do po-page-default não estou conseguindo acessar e trocá-lo

**Comment by rafaellmarques** [comment, marked as answer]
Bom dia @HandreiFont , tudo bem?
O componente `PoPageDynamicSearch` não tem um `literal` que forneça essa personalização.

Dessa forma caso queira, pode abrir uma issue de melhoria para analisarmos.
